.github/actions/build-gk/action.yaml

```diff
@@ -0,0 +1,86 @@
+name: 'build gk'
+description: 'Build GenomeKit conda package and save it under ~/conda-bld'
+inputs:
+  platform:
+    description: 'platform name used in the mamba installer url'
+    required: true
+runs:
+  using: "composite"
+  steps:
+    - name: get meta.yaml from conda-forge
+      shell: bash -l -e {0}
+      run: |
+        set -x
+        mkdir -p conda-recipe
+        curl -s -L -o conda-recipe/meta.yaml https://raw.githubusercontent.com/conda-forge/genomekit-feedstock/main/recipe/meta.yaml
+        set +x
+
+    - name: try to restore the mamba env
+      id: restore_mamba_cache
+      uses: actions/cache/restore@v4
+      with:
+        key: mamba-env-${{ runner.os }}-${{ runner.arch }}-${{ hashFiles('.github/workflows/run-tests.yaml', '.github/actions/**', 'conda-recipe/meta.yaml') }}
```

Comment:
shouldn't this regenerate on the build action rather than run-tests?

Reply:
This is a `restore` so it won't try to regenerate. I'm not sure I understood your comment correctly.

Reply:
if the build-gk github action is modified, the cache isn't invalidated

Reply:
but '.github/actions/**' is included in the hash key, so the cache *will* be invalidated.

Reply:
sorry yeah, lost track of the other deps.  it's a bit weird to specify downstream dependencies in a cache key.  what if there are new workflows?